build-system/tasks/serve.js

```diff
@@ -44,6 +44,17 @@ function serve() {
     log(colors.green('Serving unminified js'));
   }
 
+  // Get if we are offline
+  if (argv.offline) {
```

Comment:
`offline` can be easily overloaded. I propose two things wrt naming:

- prefix with `dev_dashboard`
- boolean arguments of the sort can get complex later as we'll have to map argument combinations. I suggest naming this `dev_dashboard_serve_mode` with `cdn` or `local` as possible options, defaulting to `cdn`.

Reply:
What about the name `dev_dashboard_amp_source`? As isn't their already a "serve mode" which is used for a4a and things?

Reply:
Discussed offline.

`--offline` should become `dev_dashboard_amp_source`.

Where the values are `cdn` and `serve_mode` (as serve mode refers to the locally built version of AMP as that is what the serve mode API refers to.).

E.g by default this should be `--dev_dashboard_amp_source=cdn`